tests/integration/test_join.py

```diff
@@ -381,6 +381,9 @@ def test_join_alias_w_projection(c, parquet_ddf):
     assert_eq(result_df, expected_df, check_index=False)
 
 
+@pytest.mark.skip(
```

Comment:
Should we mark this as `xfail` instead to be in line with some of the efforts made in #867